### PR TITLE
embed schema.json using go 1.16 support for static resources

### PR DESCRIPTION
**Pull request by ndeloof**
The `embed` package add support for embedding static resources without relying on code generation
https://golang.org/pkg/embed/

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,13 +16,13 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x]
+        go-version: [1.16.0]
```
Comment (chris-crone) [inline .github/workflows/ci.yml:19]: We should make this float with patch releases.

```suggestion
        go-version: [1.16.x]
```

File: schema/compose-spec.json [hunk 2]
```diff
@@ -87,7 +87,7 @@
                 "dockerfile": {"type": "string"},
                 "args": {"$ref": "#/definitions/list_or_dict"},
                 "labels": {"$ref": "#/definitions/list_or_dict"},
-                "cache_from": {"$ref": "#/definitions/list_of_strings"},
+                "cache_from": {"type": "array", "items": {"type": "string"}},
```
Comment (chris-crone) [inline schema/compose-spec.json:90]: Were the changes to the schema intentional?

Comment (ndeloof) [reply schema/compose-spec.json:90]: I used the latest from compose-spec repo

**Review by chris-crone — APPROVED** [review]
LGTM with CI fix

**Pull request merged**